### DISCUSSION TITLE
How can i use Rednet on the Wireless Pocket Computer?

### DISCUSSION THREAD

**Discussion by lightyuwu** [cc-tweaked/CC-Tweaked#1956] [Q&A] [answered]
I tried to use ```rednet.open("back")``` and ```rednet.send(3, "test")``` (3 is the ID of the normal computer with a modem which works properly) but rednet.open() does not work, and rednet.send() just returns false.

**Comment by zyxkad** [comment]
Show the screenshot of exact error please.

If `rednet.open` does not work, it means your pocket computer does not have a wireless modem on the back.
You cannot send anything before you can open on the modem.

**Comment by lightyuwu** [comment]
I can't send a screenshot right now, but when i do ```rednet.open()``` no matter the side, it always says that there isn't a modem there. And what do you mean having one on a back? I just put the modem and the pocket computer in the crafting Table

**Comment by Wojbie** [comment]
Do you have optifine installed? Can you press F3+h to activate extended tooltips and show me the pocket tooltip? Does it say "none" when you ran peripherals program? What MC and cc version are you using?

**Comment by lightyuwu** [comment]
I'm running Minecraft 1.21, Neoforge and the All The Mods 10 Version 10.0-40 Modpack, which i doubt is the problem, and it uses Sodium. The tooltip is just computercraft:pocket_computer as far as i can tell, and the data tags are just computercraft:dyable i think. And yes, it does say none when i run the peripherals program.

**Reply by Wojbie** [reply]
This would indicate that pocket does not have modem on it. Can you place ender or wireless modem in slot to right of your pocket in your toolbar and use program equip to install it?

**Reply by lightyuwu** [reply]
I can try that next time, but why is there a Wireless Pocket Computer recipe then?

**Reply by Wojbie** [reply]
The recipe should be working but it is possible one of other mods in modpack interfered with it or you have found a bug. CC is in alpha release for 1.21 after all

**Reply by lightyuwu** [reply]
Oh, okay then. I'll tell the server owner to see if i can get the Wireless Modem back.
Thanks for the Help, i'll mark this as the answer if it works tomorrow, as I can't test it as of this moment

**Reply by SquidDev** [reply, marked as answer]
I suspect this is caused by a combination of https://github.com/cc-tweaked/CC-Tweaked/pull/1888 and ed0b156e05ff64784f2f80346c82ab5142d954dc. When the pocket computer is crafted, it doesn't correctly update the peripheral on the computer. If you put the computer in a chest and leave it for 5 seconds (so it turns off), the modem should then work.

**Reply by lightyuwu** [reply]
I will try that as well tomorrow, thanks!

**Reply by lightyuwu** [reply]
Putting it in a chest and waiting worked. Thank you so much for the help!
